### PR TITLE
Fix csproj for arm64 (how did i forget this????)

### PR DESCRIPTION
**Pull request by Beyley**
# Summary of the PR
Fixes the native package csproj for ARM64 so the libs actually work (oops)

### CODE REVIEW

File: src/Native/Silk.NET.SDL.Native/build/net461/Ultz.Native.SDL.props [hunk 1]
```diff
@@ -9,9 +9,11 @@
     <_Ultz_NativePackages_SDL_NativeRuntime Condition=" '$(_Ultz_NativePackages_SDL_NativeRuntime)' == '' And '$(_Ultz_NativePackages_SDL_IsLinux)' == 'true' And ('$(Prefer32Bit)' == 'false' Or '$(PlatformTarget)' == 'x64')">linux-x64</_Ultz_NativePackages_SDL_NativeRuntime>
     <_Ultz_NativePackages_SDL_NativeRuntime Condition=" '$(_Ultz_NativePackages_SDL_NativeRuntime)' == '' And '$(_Ultz_NativePackages_SDL_IsWindows)' == 'true' And ('$(Prefer32Bit)' == 'true' Or '$(PlatformTarget)' == 'x86')">win-x86</_Ultz_NativePackages_SDL_NativeRuntime>
     <_Ultz_NativePackages_SDL_NativeRuntime Condition=" '$(_Ultz_NativePackages_SDL_NativeRuntime)' == '' And '$(_Ultz_NativePackages_SDL_IsWindows)' == 'true' And ('$(Prefer32Bit)' == 'false' Or '$(PlatformTarget)' == 'x64')">win-x64</_Ultz_NativePackages_SDL_NativeRuntime>
+    <_Ultz_NativePackages_SDL_NativeRuntime Condition=" '$(_Ultz_NativePackages_SDL_NativeRuntime)' == '' And '$(_Ultz_NativePackages_SDL_IsWindows)' == 'true' And '$(PlatformTarget)' == 'arm64'">win-arm64</_Ultz_NativePackages_SDL_NativeRuntime>
```
Comment (Perksey) [inline src/Native/Silk.NET.SDL.Native/build/net461/Ultz.Native.SDL.props:12]: does .NET 4.6.1 even support arm lmao

**Review by Perksey — APPROVED** [review]

**Comment by Perksey** [comment]
Build is failing

**Comment by Beyley** [comment]
twas just me forgetting computers arent magic fairy dust

**Comment by Beyley** [comment]
Would it be acceptable if GLFW and SDL imports a glob instead of a specific native version? we are hitting a chicken and egg scenario right now trying to update the native package version GLFW and SDL reference, which the only reasonable solution is to not update the native package version SDL and GLFW reference, and have users manually update them, but i believe you can glob versions like `3.3.*` instead of referencing `3.3.3.1`

**Comment by Perksey** [comment]
Yeah I'm fine with this

**Review by Perksey — APPROVED** [review]

**Pull request merged**